goss-testing/scripts/check_bgp_neighbors_established.sh

```diff
@@ -49,19 +49,46 @@ function err_exit
     exit $rc
 }
 
-kubectl -n metallb-system get cm metallb -o jsonpath='{.data.config}' > $TMPFILE ||
-    err_exit 10 "ERROR: Command failed (rc=$?): kubectl -n metallb-system get cm metallb -o jsonpath='{.data.config}' > $TMPFILE"
+TOKEN=$(curl -s -k -S -d grant_type=client_credentials -d client_id=admin-client -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token')
+
+# check if metalLB configmap exists
+kubectl -n metallb-system get cm metallb 2>&1 |grep -i error|wc -l > $TMPFILE ||
+    err_exit 10 "ERROR: Command failed (rc=$?): kubectl -n metallb-system get cm metallb 2>&1 |grep -i error|wc -l " >> $TMPFILE
```

Comment:
The "ERROR" here is unnecessary because the err_exit already prepends and "ERROR".  This is what I get on the error:

"ERROR: ERROR: Command failed (rc=1): kubectl -n metallb-system get cm metallb 2>&1 |grep -i error|wc -l "